README.md

```diff
@@ -8,137 +8,138 @@ Trevor Cromwell
 
 Cheghali Elkhalili
 
-## Introduction  
+## Introduction
```

Comment:
All of my README changes actually came from my markdown lint plugin that automatically fixes lint errors when I save the file. I did notice that in addition to the auto-fixed lint errors, you have several level 1 headings in your README (the `#` symbol in markdown) and best practice is to only have 1 top level heading

Reply:
Very nice citation and comment.